rotkehlchen/exchanges/binance.py

```diff
@@ -1038,12 +1217,37 @@ def query_online_deposits_withdrawals(
         )
         log.debug(f'{self.name} withdraw history result', results_num=len(withdraws))
 
+        fiat_deposits = self._api_query_list_within_time_delta(
+            start_ts=Timestamp(0),
+            end_ts=end_ts,
+            time_delta=API_TIME_INTERVAL_CONSTRAINT_TS,
+            api_type='sapi',
+            method='fiat/orders',
+            additional_options={'transactionType': 0},
```

Comment:
This was `transaction_type` but when checking their docs it's `transactionType` https://binance-docs.github.io/apidocs/spot/en/#get-fiat-deposit-withdraw-history-user_data